include/experimental/__p0009_bits/config.hpp

```diff
@@ -205,7 +205,7 @@ static_assert(_MDSPAN_CPLUSPLUS >= MDSPAN_CXX_STD_14, "mdspan requires C++14 or
 #endif
 
 #ifndef _MDSPAN_USE_CLASS_TEMPLATE_ARGUMENT_DEDUCTION
-#  if (!defined(__NVCC__) || (__CUDACC_VER_MAJOR__ >= 11 && __CUDACC_VER_MINOR__ >= 7)) && \
+#  if (!defined(__NVCC__) || (__CUDACC_VER_MAJOR__ * 100 + __CUDACC_VER_MINOR__ >= 1107)) && \
```

Comment:
No way Jose

I was going to observe that it could be confusing because we do this in Kokkos
https://github.com/kokkos/kokkos/blob/07a500982de5fd3a0876f5ef2c1a41f7130512cb/core/src/Kokkos_Macros.hpp#L110-L111
But this inconsistent with other macro guards in this project so please don't
https://github.com/kokkos/mdspan/blob/df6d31b2e6fe1269ed5770aa1f8205308b8cab2c/include/experimental/__p2630_bits/submdspan_mapping.hpp#L179